build/common-tools.mk

```diff
@@ -10,7 +10,7 @@ MKDIR = mkdir -p
 
 CC = $(GCC_ARM_PATH)$(GCC_PREFIX)gcc
 CPP = $(GCC_ARM_PATH)$(GCC_PREFIX)g++
-AR = $(GCC_ARM_PATH)$(GCC_PREFIX)ar
+AR = $(GCC_ARM_PATH)$(GCC_PREFIX)gcc-ar
```

Comment:
Regular `ar` started to complain about missing plugins. `gcc-ar` wrapper supplies `ar` with all necessary paths.